www/css/ionic.app.css

```diff
@@ -9772,10 +9772,30 @@ body {
   .image-card .mini-profile-image {
     border-radius: 40px;
     vertical-align: middle;
-    border: 2px solid #0a9dc7;
+    border: 2px solid #efefef;
     width: 40px;
     display: block;
     margin: 0 auto; }
   .image-card .profile-label {
     font-size: 14px;
     font-weight: 600; }
+
+.image-header {
+  text-align: right; }
+  .image-header .header-title {
+    position: absolute;
+    top: 10px;
+    right: 10px; }
+  .image-header h4, .image-header h5 {
+    font-family: Oswald;
```

Comment:
This file is the css auto-generated from "https://github.com/ronniebermejo/citymap/blob/master/scss/ionic.app.scss"
Add your `instImg` there instead. :v: